internal/document_sync/types.go

```diff
@@ -0,0 +1,23 @@
+package document_sync
```

Comment:
same with this, this should be in internal/lsp/document_sync_types.go in `package/lsp` or something like that